### ISSUE TITLE
Bounty Fixing Issues with the latest Version of TTRss

### ISSUE THREAD

**Issue by seraphyn** [levito/tt-rss-feedly-theme#66]
I will just add a bounty to fix the issues with the new code of tinytinyrss

**Comment by valvin1** [comment]
Added 5$ i will now try to spread the bounty

**Comment by Gravemind** [comment]
Here is the theme I created: https://github.com/levito/tt-rss-feedly-theme , I think it's good enough for me, tell me if it's good enough for you...

**Comment by Gravemind** [comment]
@seraphyn: Fair enough. I wont try to fix 2000 lines of css... I thought starting from the default theme rather than from scratch, and using Less CSS, could be more maintainable; but I'm not an active CSS developer, so only time will tell.

**Comment by jamespo** [comment]
@Gravemind I've moved to your theme & agree this approach should be more maintainable

**Comment by sdenaro** [comment]
@Gravemind I installed your theme yesterday, it's a nice improvement over the default tt-rss themes.

**Comment by levito** [comment]
Hi all! First of all, thanks for setting up the bounty - I was never aware of how much people value this theme and how much it is actually used - wow!

@Gravemind, thanks for putting your theme together - I really like it! I also think that it would not make a lot of sense to fix my theme. My approach would also be to create a new theme using Less or Stylus in order to build on top of the current tt-rss base theme or the dojo flat theme.

I just won't have a lot of time to do so, at least not this year. I'll see if I find some time next year, and I can imagine to collaborate on this with you, @Gravemind - what do you think?

For now, I'd suggest using the feedlish-theme - it really comes pretty close!

**Comment by Gravemind** [comment]
Thank You @levito for your awesome work and support. Of course I'll gladly welcome Issues and PRs on feedlish.

For now I'd like to see how close feedlish can get to feedly with the less code possible, for maintainability. But it might become irrelevant even before I get it to the "close enough" I'm looking for... Then if the consensus would really prefer a perfect-feedly anyway, I'd rather follow the move, especially if the maintenance can be shared.

**Comment by valvin1** [comment]
@Gravemind I tested your theme too early but after upgrading it yesterday I've seen really good improvments I like :) Nice job !

**Comment by levito** [comment]
Hey guys, please have a look at the recent changes!

Inspired by @Gravemind's work (thank you!) I decided to do a fresh start.

I'd like to know if you're happy with the current state, and if not, what's missing in your opinion.

Thanks a lot for your patience!

**Comment by x86dev** [comment]
@levito The screenshots in the README.md still represent the "old" state, right? If so, could you please also update those? Thanks!

**Comment by vinzv** [comment]
First of all: thank you so much for making TTRSS useable again!

For me so far everything is working and I'd accept the bounty claim.

**Comment by seraphyn** [comment]
I accept it, too now

**Comment by valvin1** [comment]
I accept too.
Thank you
